src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -132,14 +132,19 @@ protected virtual void OnCustomNodeRemoved(Guid functionId)
         /// <param name="isTestMode">
         ///     Flag specifying whether or not this should operate in "test mode".
         /// </param>
+        /// <param name="libraryServices">LibraryServices used for code block node initialization.</param>
```

Comment:
is this all it's used for..?

Reply:
@mjkkirschner Yes it is. It used to be wrapped and used in NodeLoader but since we changed how node info are deserialized, we will need it from upper level somehow to support CBN creation.